cg/cli/upload/scout.py

```diff
@@ -162,28 +161,16 @@ def upload_case_to_scout(context: CGConfig, re_upload: bool, dry_run: bool, case
 @click.command(name="rna-to-scout")
 @click.option("--dry-run", is_flag=True)
 @click.option("-r", "--research", is_flag=True, help="Upload research report instead of clinical")
-@click.option(
-    "-u",
-    "--update-fusion-report",
-    is_flag=True,
-    help="re-upload existing fusion report",
-)
```

Comment:
Why is removing this the solution?

Reply:
With the new `scout load report` command, this update flag is no longer available

Reply:
So scout has a new function, which removed an option for us to use?

Reply:
Yes! The new function replaces by default:
https://github.com/Clinical-Genomics/scout/blob/main/scout/load/report.py

Reply:
the cli call defined in:
https://github.com/Clinical-Genomics/scout/blob/main/scout/commands/load/report.py#L85